src/components/Annotation/index.tsx

```diff
@@ -26,6 +26,7 @@ const Annotation: React.FC<Props> = ({
     x={xScale(data[0])}
     y={0}
     height={height}
+    // @ts-ignore
```

Comment:
should we approach this?

Reply:
Dont remember whats the problem but absolutely